cmd/kubeseal/main.go

```diff
@@ -108,6 +101,28 @@ func init() {
 	flag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 }
 
+func initUsualKubectlFlags(flagset *flag.FlagSet) *clientcmd.ConfigOverrides {
+	overrides := clientcmd.ConfigOverrides{}
+	kflags := clientcmd.RecommendedConfigOverrideFlags("")
+	clientcmd.BindOverrideFlags(&overrides, flagset, kflags)
+	return &overrides
+}
+
+func initClientFromFlags() clientcmd.ClientConfig {
+	return initClient(*kubeconfig, *globalOverrides, os.Stdout)
+}
```

Comment:
What are we expecting to read from `stdout`? I've noticed the original code reads from `stdin`, why this change?

Reply:
This is a mistake. Will fix it

Reply:
Done